script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_4_1.txt

```diff
@@ -5,16 +5,16 @@ I think back to the incident with my father, Tohno Makihisa.
 Makihisa was a strict father, but at the same time, he was a quiet man who didn't show his emotions.
 <Page3>
 Do not go out, do not go into the city, people are born with different statuses, protect our noble bloodline which should not be viewed as the same as the others―――
-This was the rule of Makihisa, and by extension, of the Tohno family.
+Those were the rules of Makihisa, and by extension, of the Tohno family.//Shiki just finished listing off a bunch of things that Makihisa didn't want him to do, unless "rule" is used in the sense of dominion over something. If it isn't used in the sense of dominion, then "rule" could also be changed with "principles"
 <Page4>
-I rebelled against that rule.
+I rebelled against those rules.//if my previous suggestion is accepted then please change this
 I felt that my father's teachings were wrong.
 Not because of a mistake in logic, but because of a lack of persuasiveness.
-...I'm not so sure now, but as a child, it felt like it didn't even belong in this era.//ruby dots <も|・><う|・><時|・><代|・><じ|・><ゃ|・><な|・><い|・>
+...I'm not so sure now, but as a child, it felt like it didn't even belong in this era.//ruby dots <も|・><う|・><時|・><代|・><じ|・><ゃ|・><な|・><い|・>, for italics
```

Comment:
I believe he's talking about Makihisa's teachings so the 2 its need to be changed.

I felt like they didn't even belong